src/libs/ReportActionsUtils.ts

```diff
@@ -700,8 +700,8 @@ function getAllReportActions(reportID: string): ReportActions {
 function isReportActionAttachment(reportAction: OnyxEntry<ReportAction>): boolean {
     const message = reportAction?.message?.[0];
 
-    if (reportAction && 'isAttachment' in reportAction) {
-        return reportAction.isAttachment ?? false;
+    if (reportAction && (reportAction.attachmentInfo ?? reportAction.isAttachment)) {
```

Comment:
Please keep the logic in other cases like before

Reply:
Waht do you think about this suggestion?

```
if (reportAction && ('isAttachment' in reportAction || 'attachmentInfo' in reportAction)) {
        return reportAction.isAttachment ?? reportAction.attachmentInfo ?? false;
    }

```

Reply:
> However for videos the attachementInfo isn't present, and isAttachement is false so it will return false.

@Tony-MK  Yes, what problem with this? In the old logic It also return false if isAttachement is false

Reply:
Sorry, Yeah. I think you are right.